website/src/components/home/CanvasList.jsx

```diff
@@ -157,6 +163,11 @@ class CanvasList extends React.Component {
+jmHhzmnWIx8VWpnvRagbAA4KaLfEipTdkLxAiSeeqns2SY6nKuWVsBXV+Jb7c1paGNaVOHJ7yFVqnFQ6Nd+r+hLRVhRUK0qK1uwVeyFWpJxObklKOZRTNTBCg2kKNI28pnOSlKallazyoIlIhACyhIhAxVvXXVmmOGHT+lgLRuitEt5EevoqPIjcC/wAd1M1DVxzz1VimSCX6huHUnHyWbWd34qxRr7QiccjyH4zWFo6MWR2ivhJyxOHjO5Mp22m1sveBPjr7aJLRZfnfSXGNY16qnX+EqTsWbcgfbtEjzzVqpiUWy9Tvylk36uanpXvRP+6ehJ6rBZ8L04g7TXdZO6IUTvhtw+2s5o5TmOJx0Vnpx+zRCEqNm131T0EDiQqQvSi790LPZ8M/73l8zoRu0nOE0/DtFoiHTpDiDyR6cfsjKEu2i4+2Cc8OGHRaFIFzY0jpHvksel8P0gMSXHmYB4cVsUKJa2McoxVcml0Z+LTGVzLQdYHDcqtI58O/RS3jWGDRpAKga/ioroJO2WqAkgRmY76LT2pc7mVn3S2agwy/K0GDEnirIfJPCpN/iOISNqlOdimhqnaaNjxTjLRNiontUgqJpco2jQscvszbZKLECrlRgKWnShWqa4nPyeNNZUw2ShTYb0KHJGr08hyoSIRK1nmAlKmQnIGKhIhACq/crNp5G9h8RiFnlafw+f8AMORHkoZPYyzF70OruUDa8cj/AErt6UCHFwGeaoikIWA3mjQrSM93ZVulWj6hr7LPstHx7z3K0ymcvJRvZbFj7VawcHCevP271zH3uGmNioRx2SI4GcloGyE+qa6wKcZpE+U10Z5vcuP006mOsAYYakzuQ6rhgInP2V1lj4Ybkx9iz0Q5g5TfZBQdlwUltrkDDA+mOvRObSjCFn258GN+ftKitsqlKkUtomSf6UgOQ3xCYDGHYU1ip7Tsu/RWFLNu46UQTqFJT3qWyN2RJwABM7sE0twCUXdnR8KK5bAOSlyrOepWlSa0dCE4ym9jK1SEUakqR7E0NhStcSnhN5bvRK0ocVD8xLtSo1SLeSlkoSUIQoms52UEpJSLoHgxyE1KUDFRKagoAWUtjvVtO0UqeZc8Bx3B2A6yQqtutIpsJ105rk22hxL6k4ggg8QS70CTjaaJQdOz2K3MDhnPn3osZrNDpvVu7LwFopMqtP3NxGoOo6GVVtw2TK51PpnRb+US0KsEY4T6yPz5Ldoua4AyMp5rljVkZ9wls9vLZE/12UOFiU6OxpFh9t3NTFrY3+Hcrkf9T1DlYZfO9w068OSj6bLVkOjdTZwx1VSuaYWLVvcDGfBZ1ovicA7ph2E+DYPIka9ptLWg7zgB33gubqPBJ77z/ChtN4z9M+HNQGrvVsYUZ5ZLLZxxPffqtW7W9+ne5YbK0kDNb1hIIiPHvuVGWkShtly9a3y7LVdupujwjDxXB3d8TVKNQU6hLqbgCJOLQdx3TOC6L44t2xZvlg41HBv/AGiXE/8Ar5riLys+1SDx9zMemq0ePFcdlWbLKE1xdHdm2bWIMgqzSti4/wCHbbLQCeXBbsq5wXRXDyckXaZsfrFE+2LMlEqPpxLX5+X7LbrUZU1O2KgCo3FPinoqj5OSMuVmv+tSrF2kI9JF3/fk+xyE1zoxJhVqlvYNZ5KZgLaJWTWvf/a3qT6KhaLwe79xHLAJ0M6NzwMzHVUa16NBhv1Hfp/KwCCcyT6KzQZARQIS+LUSMSqVio/4Z1cSfbmkvh2Cs2Zv+Fn/ABH4TrQGp8EXmWOdQJjEuZuO8ec+K7OuQ9pI101XlFoc5hD2mHMIIPfRejXDeDbTRDxAI+8Tk6PxGIKyZ4U+SNuCdriULY91PGCR3/Cr/q21AQD6Eey6C02eRj+Asi1XQ0kxgeBg6a7lXGS+RygzPqh4+13Q+/8ACoVbRWBP0npirdssNanJb/kaMwPuHCNfJZbr2IMHaaZ1EK1K+tkGSC01jo7qnUi/9zo5Yn+PNVKl58yfBLZKFauYaCBvgwOqlX3oiW320NEDvmks5fUOULTsVwNZE/WcJ18Ny1qNjaNOx/aqc0uiccbfZXsFlLYkGe/NbYrBo/ACgDIy9uwoLdaBTY57jg0E+GipdyZoS4o5b4rtpfWDJxaJPN0QPCPFLSyAO5ZdAGo41H5vJJ78PBaJfEroRjxSRzskuUrKdzGJG4/gwuqs1baHHVcldRwnetmz19kqTIGzKFXo2oHDI95KeUgHhMclBTXIASUJEIA5upWLsyTzUZcmlBUgsa4oaEbKe0JiH02qeVG0YJXO777xSGZV8Ow73rRszf8AEz/iPx/KyL4Potprw2mCTADR+E2CM+q3PinfDd7OstaT9hweN439M1SfeYLj9P088ee5XrJYmVTjOUiNQozWt9E8cvy0epMLXtDmkEESDpEYKF7N/iqfwi1jGii5ztn9pnInTLJdQ65gf3Hy/pcuTSZ047Rh/KB7hZ143Ky0QCwk5bQwI6+i6wXU1uhPX0/tDqcYZfhCy10DxnB0PgoUztPJeMwMR471p06MYAQBkMguqDO+yo3WNj/u8QPVDyuXYLGl0c+ympWMC1X3KNHHqJSOuuP3+XslyRJRoy3mFx/xbb9twoNOAxqfkN47/BdZfc0KRcCC79oOp9l51TaZLnfc4yTvcTiVp8aFvl9GbyJ0qLFBsfhNvGrFMnXLvzUjMMFQvZ/2t4ytpgJrtGAWmCs+w5K7KGIdUensvJ7YxkHeooUVQCOSBmxRvVpzBH4VtlQOyMrm2FT03xiClQWbyFkfrH70IoRlpEIUgFT2pUJMCRiRyEIAw711Whef/QHIfgJUJsDniuiuHNv/ABQhGX2ksXuOxu77hzH5Xo7MhyQhcjJ2dfH0Oq5qtVySoVSGVNyvUckIQP5JzqqNp1QhAzjvjPJvJy4NunVCF0vF9rOd5XuJxn0WXef3jokQtKMpfsGStFCEAPGShfk7qlQhCGhStQhAEqEITA//2Q==";
```

Comment:
Just looked at the actual page and I now think Artur should be responsible for reviewing this.